Source/JTSAnimatedGIFUtility.m

```diff
@@ -104,13 +104,24 @@ static void releaseImages(size_t const count, CGImageRef const images[count]) {
     int delayCentiseconds[count]; // in centiseconds
     createImagesAndDelays(source, count, images, delayCentiseconds);
     int const totalDurationCentiseconds = sum(count, delayCentiseconds);
-    NSArray *const frames = frameArray(count, images, delayCentiseconds, totalDurationCentiseconds);
-    UIImage *const animation = [UIImage animatedImageWithImages:frames duration:(NSTimeInterval)totalDurationCentiseconds / 100.0];
-    releaseImages(count, images);
-    return animation;
+    UIImage *image = nil;
```

Comment:
Looks like `dev` and `master` were out of date. I've updated `dev`. Would you please rebase this against latest dev so the merge is a fast-forward?